simple_cb/simple_cb_magma.py

```diff
@@ -41,12 +41,15 @@ def __init__(self, num_tracks, width):
                 self.wire(reg.ports.O, self.read_config_data_mux.ports.I[idx])
                 # Wire up config register resets
                 self.wire(reg.ports.reset, self.ports.reset)
+                # Connect config_en for each config reg
```

Comment:
Let's add this logic down on line 66. That way we don't need it in 2 places (i.e. remove line 52)